### ISSUE TITLE
value! generates an unused import warning

### ISSUE THREAD

**Issue by wesleywiser** [Geal/nom#224]
```rust
named!(
    pub one<&[u8], u8>,
    value!(
        1,
        char!('1')));
```

Results in:
```
<nom macros>:4:5: 4:25 warning: unused import, #[warn(unused_imports)] on by default
<nom macros>:4 use $ crate:: HexDisplay ; match $ submac ! ( $ i , $ ( $ args ) * ) {
                   ^~~~~~~~~~~~~~~~~~~~
<nom macros>:32:1: 32:34 note: in this expansion of value! (defined in <nom macros>)
src/lib.rs:59:1: 63:22 note: in this expansion of named! (defined in <nom macros>)
```

**Comment by nanotech** [comment]
It's this line: [src/macros.rs#L537](https://github.com/Geal/nom/blob/29273103abf4208f7e93bc06b974c9eaa0d1ea52/src/macros.rs#L537). I tried removing it and all tests still pass. Is it a debugging artifact?

**Comment by Geal** [comment]
Yup, I forgot to remove it.

Fixed in fd4afa8
